kolibri/utils/conf.py

```diff
@@ -47,6 +47,11 @@
         )
     os.mkdir(KOLIBRI_HOME)
 
+# Create a folder named logs inside KOLIBRI_HOME to store all the log files.
+LOG_ROOT = os.path.join(KOLIBRI_HOME, "logs")
+if not os.path.exists(LOG_ROOT):
+    os.mkdir(LOG_ROOT)
```

Comment:
Just to double check, this should happen before we do anything with options that might trigger its logger, right?

Reply:
yes, because we set up some basic logging in the options I believe.